va/compat_win32.h

```diff
@@ -46,6 +51,15 @@ typedef LONG NTSTATUS;
 
 typedef unsigned int __uid_t;
 
+#if _MSC_VER
+#define getenv _getenv
+inline char* _getenv(const char *varname)
+{
+    static char _getenv_buf[32768];
```

Comment:
I am not familiar with windows GetEnviomentVariableA, but looks it is too large?

Reply:
According to [the API documentation](https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getenvironmentvariable) for the `lpBuffer` argument to `GetEnviomentVariableA`, "An environment variable has a maximum size limit of 32,767 characters, including the null-terminating character".

As [getenv](https://cplusplus.com/reference/cstdlib/getenv/) returning char* is not freed by the caller and points to an internal block of memory whose content or validity may be altered by further calls to getenv, I wanted to follow this behavior with the wrapper function here (as opposed as returning dynamically allocated memory the caller does not free).

Reply:
Sounds good to me. @XinfengZhang ?